CHANGELOG.md

```diff
@@ -9,14 +9,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+* Added `Plane.offset`
```

Comment:
looks like a merging issue because this same line is already present a few lines later.